sdk/keyvault/azure-keyvault-keys/CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Release History
 
+## 4.7.0 (2022-09-13)
+
+### Breaking Changes
+- Clients verify the challenge resource matches the vault domain. This should affect few customers,
+  who can provide `verify_challenge_resource=False` to client constructors to disable.
+
+### Other Changes
+- Changes from version 4.7.0b1 have been reverted and will be included in version 4.8.0b1
+
+## 4.7.0b1 (2022-08-12)
```

Comment:
Was this released from another branch or something?

Reply:
4.7.0b1 was released from `main` after 4.6.1 was released, and these changes are being applied on top of 4.6.1's release tag. So I just had to pull over the changelog entry from the beta release to make the timeline consistent -- after this minor version is released, I'll release 4.8.0b1 that merges the changes here with the updates that were made in 4.7.0b1

Reply:
Is the target hotfix/keyvault-verify branch branched from your last GA then?

Reply:
> Is the target hotfix/keyvault-verify branch branched from your last GA then?

Yep!